docs/book/src/user/quick-start.md

```diff
@@ -45,7 +45,7 @@ a target [management cluster] on the selected [infrastructure provider].
 
    [kind] is not designed for production use.
 
-   **Minimum [kind] supported version**: v0.14.0
+   **Minimum [kind] supported version**: v0.15.0
```

Comment:
Let's keep the kind version the same while still bumping to v1.25 like you did below